egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -225,6 +225,7 @@ def train_one_iteration(dir, iter, srand, egs_dir,
                         frame_subsampling_factor, truncate_deriv_weights,
                         run_opts,
                         dropout_edit_string="",
+                        use_multitask_egs=False,
```

Comment:
Can you please revert any changes to chain-related code as long at this is not actually supported?  I think this is just confusing.